packages/notification-center/src/hooks/useRemoveAllNotifications.ts

```diff
@@ -0,0 +1,35 @@
+import { useMutation, useQueryClient, UseMutationOptions } from '@tanstack/react-query';
+import type { IMessage } from '@novu/shared';
+
+import { useNovuContext } from './useNovuContext';
+import { useFetchNotificationsQueryKey } from './useFetchNotificationsQueryKey';
+
+interface IRemoveAllNotificationsVariables {
+  feedId?: string;
```

Comment:
If we make this optional then the mutation code should be prepared for it, checking if the `feedId` is being set and executing the mutation if so (line 23).